scripts/update-apk.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e
+
+git config --global user.name "Travis CI"
+git config --global user.email "noreply+travis@fossasia.org"
+
+export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
+export PUBLISH_BRANCH=${PUBLISH_BRANCH:-master}
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/phimpme-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+	echo "We upload apk only for changes in development or master, and not PRs. So, let's skip this shall we ? :)"
+	exit 0
+fi
+
+
+git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossasia/phimpme-android apk > /dev/null
+cd apk
+
+if [[ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]]; then
+	/bin/rm -f *
+else
+	/bin/rm -f phimpme-dev-*.apk
+fi
+
+\cp -r ../app/build/outputs/apk/app-debug.apk app-debug.apk
+\cp -r ../app/build/outputs/apk/app-release-unsigned.apk app-release-unsigned.apk
+
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+	for file in app*; do
+		if [[ $file = "phimpme"* ]]; then
+			continue
+		fi
+		mv $file phimpme-master-${file%%}
+	done
+fi
+
+if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+	for file in app*; do
+		if [[ $file = "phimpme"* ]]; then
+			continue
+		fi
+		mv $file phimpme-dev-${file%%}
+	done
+fi
+
+# Signing Apps
+
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+    echo "Push to master branch detected, signing the app..."
+    cp app-release-unsigned.apk app-release-unaligned.apk
+    jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
+    ${ANDROID_HOME}/build-tools/26.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
```

Comment:
Wrong command

Reply:
@iamareebjamal could you please tell me what's wrong with the command here?